aten/src/ATen/native/mps/operations/BinaryOps.mm

```diff
@@ -181,6 +181,10 @@ void div_mode_template(const Tensor& self, const Tensor& other,
     TORCH_CHECK(self.scalar_type() != ScalarType::Long,
                 "MPS: does not support floor_divide op with int64 input");
   }
```

Comment:
>*rounding_mode
Is there no .type or .name or something ?

Reply:
Its a string:
```
   c10::optional<c10::string_view> rounding_mode,
```